tools/reg_parse.c

```diff
@@ -117,10 +119,12 @@ pre_parse( char * dir, FILE * infile, FILE * outfile )
     for ( p = inln ; ( *p == ' ' || *p == '	' ) && *p != '\0' ; p++ ) ;
     if ( !strncmp( p , "include", 7 ) &&  ! ( ifdef_stack_ptr >= 0 && ! ifdef_stack[ifdef_stack_ptr] ) ) {
       FILE *include_fp ;
-      char include_file_name_local_registry[128] ;
-      char include_file_name[128] ;
+      char include_file_name_local_registry[MAX_PATH] ;
+      char include_file_name[MAX_PATH] ;
       p += 7 ; for ( ; ( *p == ' ' || *p == '	' ) && *p != '\0' ; p++ ) ;
-      if ( strlen( p ) > 127 ) { fprintf(stderr,"Registry warning: invalid include file name: %s\n", p ) ; }
+      fprintf( stderr, "Len : %d\n", strlen( p ) );
```

Comment:
Was this print statement added for debugging purposes? If so, perhaps it could be removed.

Reply:
Thanks for the catch!